statsmodels/sandbox/tests/test_predict_functional.py

```diff
@@ -31,6 +31,7 @@ def teardown_class(cls):
         if pdf_output:
             cls.pdf.close()
 
+    @dec.skipif(not have_matplotlib)
```

Comment:
is this one necessary?
I thought only test function and methods need it.  `test_xxx` as identified by nose